packages/linear-progress/src/mwc-linear-progress-base.ts

```diff
@@ -85,7 +85,11 @@ export class LinearProgressBase extends BaseElement {
       ...addHasRemoveClass(this.mdcRoot),
       getPrimaryBar: () => this.primaryBar,
       getBuffer: () => this.bufferElement,
-      setStyle: (el: HTMLElement, property: string, value: string) => el.style[property] = value,
+      setStyle: (el: HTMLElement, property: string, value: string) =>
```

Comment:
can we update the type of property here, instead of casting at the use site?

Reply:
This needs to match an interface defined in material components web. We can update that too, though the one place that calls this interface function calls it with `-webkit` prefixed strings, so then we need a cast there anyway. Maybe we should augment `CSSStyleDeclaration`. I just added a TODO for now, in the interest of incremental progress. Also pointed out what those two excluded properties are.

Reply:
+1, thanks for the comment